runner/wellknown_recipes_test.go

```
@@ -0,0 +1,129 @@
+	"github.com/stretchr/testify/require"
+)
+
+var httpxBinary string
+
+func TestMain(m *testing.M) {
+	tmp, err := os.MkdirTemp("", "httpx-recipes-*")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create temp dir: %v\n", err)
+		os.Exit(1)
+	}
+	defer os.RemoveAll(tmp)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)